src/css/tabs/receiver.css

```diff
@@ -641,6 +646,21 @@
     margin: 0;
 }
 
+.tab-receiver .elrsContainer {
+    float: left;
+    width: calc(100% - 20px);
+}
+
+.tab-receiver .elrsUid {
+    float: left;
+    margin-left: 5px;
+    width: calc(100% - 20px);
+}
+
+.tab-receiver .elrsPassphrase {
+    margin-left: 5px;
+}
+
```

Comment:
Why using floats here? There are far better constructs like `flex` now days, might be better to use for new code.

Reply:
@chmelevskij flex would be better, but I generally try to match the code I'm changing, and this particular tab uses floats. How strongly do you feel about it? I could convert it if you feel really strongly about it

Reply:
makes sense thanks 🙏

Reply:
At some moment we need to get out all this float. They hurt us when we try to make the application valid for RTL languages. But I suppose the moment is not now ;)